src/aws-api-mcp-server/awslabs/aws_api_mcp_server/core/security/mcp-security-policy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Customers will not have access to this file, we need an approach where they can provide their security policy.